modules/hooks/init.js

```diff
@@ -774,5 +774,10 @@ export default function() {
 
     // Keep a list of actors that need to prepareData after 'ready' (generally those that rely on other actor data - passengers/mounts)
     game.wfrp4e.postReadyPrepare = [];
+
+
+    loadTemplates({
+      aspectDetails: 'systems/wfrp4e/templates/items/partials/item-aspect-details.hbs'
+    });
```

Comment:
I don't see a reason to have this separate from the rest?

Reply:
Yes, you can pass an object and then call them like this:

```hbs
{{> aspectDetails}}
```
which I find easier to manage, especially in case you want to change paths to templates ;) 

However I used "dynamic partials", which can be done two ways:

1. `getPartial` is a function
```hbs
{{> (getPartial)}}
```
2. `somePartial` is a field on object
```hbs
{{> (lookup . 'somePartial')}}
```